packages/example-completeness/src/AttributeNormalizer.js

```diff
@@ -1,69 +1,52 @@
-import _ from 'lodash';
-import jp from 'jsonpath';
+const _ = require('lodash');
+const jp = require('jsonpath');
 
 const attributeTypes = [
   // @TODO: Add nested type
   {
     name: 'set',
-    identify: attribute => {
-      if (attribute.value && Array.isArray(attribute.value)) {
-        return true;
-      }
-    },
+    identify: attribute =>
+      !!(attribute.value && Array.isArray(attribute.value)),
```

Comment:
This is due to linting and a consistent return otherwise these functions return either a Boolean or undefined which is not really cool.